sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/models/KeyVaultSetting.java

```diff
@@ -56,27 +44,27 @@ public String getName() {
     }
 
     /**
-     * Get the value of the {@link KeyVaultSetting} as a {@link Boolean}.
+     * Gets the value of the {@link KeyVaultSetting} as an {@link Object}.
      *
-     * @return The value of the {@link KeyVaultSetting} as a {@link Boolean}.
+     * @return The value of the {@link KeyVaultSetting} as an {@link Object}.
      */
-    public Boolean asBoolean() {
+    public Object getValue() {
```

Comment:
Is this method needed?

Reply:
I needed to convert a generated `Setting` to `KeyVaultSetting` for the `getSetting()` and `getSettings()` operations. I could not find a better way to do it for now, as making the older constructor package private would not let me use it in the clients. There may be something I haven't thought of to expose something like that though.

Reply:
Nevermind, found a way to get rid of it :)